packages/react-i18n/src/i18n.ts

```diff
@@ -190,6 +203,27 @@ export default class I18n {
   getCurrencySymbolLocalized(locale: string, currency: string) {
     return getCurrencySymbol(locale, {currency});
   }
+
+  private humanizeDate(date: Date, options?: Intl.DateTimeFormatOptions) {
+    const today = new Date();
+
+    let humanizedDate: string;
+    if (
+      isCurrentMonthAndYear(today, date) &&
+      today.getDate() - date.getDate() === 0
+    ) {
+      humanizedDate = this.translate('today');
+    } else if (isYesterday(date)) {
+      humanizedDate = this.translate('yesterday');
+    } else {
+      humanizedDate = this.formatDate(date, {
+        ...options,
+        ...dateStyle[DateStyle.Humanize],
+      });
+    }
+
+    return humanizedDate;
```

Comment:
we don't need to store the `humanizedDate` to return it here, we can just return it immediately.